src/core/tests/Test_YumPackageManager.py

```diff
@@ -418,7 +418,7 @@ def test_ssl_certificate_issue_type1_fix_success(self):
         package_manager = self.container.get('package_manager')
         self.assertTrue(package_manager)
 
-        output = package_manager.invoke_package_manager(package_manager.yum_check)
+        output, code = package_manager.invoke_package_manager(package_manager.yum_check)
```

Comment:
Do we have/need to make the same change, in Test_ZypperPackageManager.py?

Reply:
Yes, but Test_ZypperPackageManager.py doesn't use the output of `invoke_package_manager` anywhere so no changes are needed.